kura/org.eclipse.kura.web2/src/main/resources/org/eclipse/kura/web/client/messages/Messages.properties

```
@@ -360,7 +360,7 @@ netIPv4ToolTipStatusUnmanaged=Interface is not managed by Kura
 netIPv4ToolTipStatusL2Only=Only Layer 2 is configured
 netIPv4ToolTipStatusEnabledLAN=Interface is used to create a local network.  Use this for creating an access point that other devices can connect to.
 netIPv4ToolTipStatusEnabledWAN=Interface is enabled for use as a gateway to an external network. Use this to connect to a router, wireless access point, or cellular network.
-netIPv4ToolTipPriority=Priority used to determine which interface select as primary WAN.<br><br>In case the primary WAN interface loses connection, then the next highest priority interface is selected.<br><br>Lower values correspond to higher priority.<br><br>If this field is set to an empty value or to -1, it means that the metric is chosen automatically based on the device type.
+netIPv4ToolTipPriority=Priority used to determine which interface select as primary WAN.<br><br>In case the primary WAN interface loses connection, then the next highest priority interface is selected.<br><br>Lower values correspond to higher priority.<br><br>If this field is set to an empty value or to -1, it means that the metric is chosen automatically based on the device type.<br><br>Note that when an interface is selected as WAN according to its priority, also the corresponding DNS servers will be used. 
```

I have to admit that this is quite hard to understand... May I suggest something like:

```suggestion
netIPv4ToolTipPriority=Priority used to determine which interface select as primary WAN.<br><br>In case the primary WAN interface loses connection, then the next highest priority interface is selected.<br><br>Lower values correspond to higher priority.<br><br>If this field is set to an empty value or to -1, it means that the metric is chosen automatically based on the device type.<br><br>Note that WAN priority also affects DNS servers selection. Therefore, when an interface is selected according to its WAN priority, the associated DNS server will be used.
```